File: barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/barbeariainclusao/src/main/java/br/cefetmg/barbeariainclusao/model/Usuario.java
package br.cefetmg.barbeariainclusao.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class Usuario {
    
    private Long idUsuario;
    private String nomeUsuario;
    private String telefoneUsuario;
    private String senhaUsuario;
    private String emailUsuario;
    private String fotoUsuario;
}

File: BarbeariaEInclusao/src/app/model/usuario.ts
export class Usuario {
    // Usar 'idUsuario' e 'number' (embora o backend use Long, number é o tipo padrão no TS)
    idUsuario: number; 
    
    // Suporte a campos do backend
    nomeUsuario: string;
    senhaUsuario: string;
    emailUsuario: string;
    telefoneUsuario: string = '';
    fotoUsuario?: string;
    
    // Descrição foi removida no backend, mas mantida como opcional se necessário futuramente.
    // descricaoUsuario?: string; 

    constructor() {
        this.idUsuario = 0;
        this.nomeUsuario = '';
        this.senhaUsuario = '';
        this.emailUsuario = '';
        this.fotoUsuario = '';
        this.telefoneUsuario = '';
        // this.descricaoUsuario = '';
    }
}
